lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -107,6 +107,10 @@ function dosomething_helpers_preprocess_partners_vars(&$vars) {
       if (!empty($image_node->field_image_landscape)) {
         $img = dosomething_image_get_themed_image($image_node->nid, 'landscape', '550x300');
         $vars['partner_info'][$delta]['image'] = $img;
+      } else {
```

Comment:
shouldn't `dosomething_image_get_themed_image()` then return the no image placeholder then?
it shouldn't be necessary to call `dosomething_image_get_themed_no_image()` each time an image is requested, that should happen within the image module.